### DISCUSSION TITLE
VS Code theme that distinguishes different levels of headers?

### DISCUSSION THREAD

**Discussion by hao-hong** [quarto-dev/quarto-cli#8964] [Q&A]
### Description

As the title says, I am using quarto in VS code. Interestingly, I have not found a theme (dark themes preferred but not necessary) that distinguishes different levels of headers--I mean, making different levels of headers in different colors. I am open to tools other than VS code but do not want something that has a steep learning curve. Thank you!

**Comment by mcanouil** [comment]
That's not up to Quarto.
That's VSCode behaviour. You can see the exact same thing for Markdown document.
There is nothing to be done here by either Quarto CLI (independent of your IDE) and VSCode extension.
